client/Components/App.test.js

```diff
@@ -14,3 +14,11 @@ describe("<App />", () => {
 //     const wrapper = shallow(<App />).dive();
 //   });
 // });
+
+// describe("App", () => {
+//   it("componentDidMount should fetch, and put resturants in state if ok", async () => {
+//     const renderedComponent = await shallow(<App />);
+//     await renderedComponent.update();
```

Comment:
can also try done() if you are still having async issues. syntax can be found in the documentation if needed, but both works

Reply:
Thank you